src/applications/pensions/PensionsApp.jsx

```diff
@@ -4,6 +4,8 @@ import PropTypes from 'prop-types';
 
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import { useFeatureToggle } from 'platform/utilities/feature-toggles';
+import IntentToFile from 'platform/shared/itf/IntentToFile';
```

Comment:
This should be `@department-of-veterans-affairs/platform-shared/itf/IntentToFile`